File: src/entity/role.ts
export class Role {
	public id: number | null = null;
	public account_id: number | null = null;
	public role_id: string = '';
	public server: string = '';

	constructor(roleId = '', server = '') {
		this.role_id = roleId;
		this.server = server;
	}
}

File: src-tauri/src/entity/role.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize)]
pub(crate) struct Role {
    pub(crate) id: Option<i32>,
    pub(crate) account_id: Option<i32>,
    pub(crate) role_id: String,
    pub(crate) server: String,
    pub(crate) last_update_time: Option<String>,
}
